tests/Bridge/Symfony/Bundle/Test/ApiTestCaseTest.php

```diff
@@ -160,6 +160,10 @@ public function testAssertArraySubsetDoesNothingForValidScenario(): void
 
     public function testFindIriBy(): void
     {
+        if (\PHP_VERSION_ID >= 80000) {
```

Comment:
Why?

Reply:
This causes the same error with Doctrine (incompatible method signatures). So, for now, I'm just skipping these tests on PHP 8, otherwise, the test suite fails with a fatal error and we can't see the rest of the test runs and if it passes.

Reply:
Ok, can you at least add a `FIXME` or something?